tests/integration/v3_lease_test.go

```diff
@@ -142,6 +143,86 @@ func TestV3LeaseGrantByID(t *testing.T) {
 	}
 }
 
+// TestIssue12535 ensures restarted member can recover negative leaseID from backend and handle it correctly
```

Comment:
lgtm and thanks @chaochn47 but a small suggestion, can we find a better name for the test? Something like TestV3LeaseNegativeID. The issue title or initial description doesn't give any idea about its relationship with this test unless digging deeper, and in general, always good to have more meaningful name. For easy reference to associated issue, it's fine to put issue number in the doc comment. What do you think? It's okay to change it separately if you prefer (with clean sweep. Thanks!

Reply:
Yeah, I will rename the test name and provide more clear test description with comments. 

I need to squash the commits regardless :) @spzala

Reply:
:) Awesome, thanks @chaochn47